pilot/pkg/networking/core/v1alpha3/configgen.go

```diff
@@ -67,6 +71,12 @@ func (configgen *ConfigGeneratorImpl) BuildSharedPushState(env *model.Environmen
 	wg.Wait()
 
 	configgen.PrecomputedOutboundClusters = clustersByNamespace
+	for _, p := range configgen.Plugins {
+		if p.OnPreComputeMixerPerRouteFilterConfig(env, push) != nil {
+			configgen.MixerPerRouteFilterConfig = p.OnPreComputeMixerPerRouteFilterConfig(env, push)
```

Comment:
this should be assigned to map entry
PerRouteFilterConfig[p.Name] = ..

Reply:
done